unfoldingWord/create-app
# Added Authentication

## Test Instructions

- Open https://deploy-preview-21--create-app.netlify.app/
- Should see a login form
- Log in using your door43 credentials
- refresh the page, the login should persist
- Open the menu drawer by clicking the hamburger icon 
- Log out by clicking the logout button at the bottom of the drawer menu 
- Should render the login screen


This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword/create-app/21)
